### PR TITLE
Chinese Translation for Resources Section

### PR DESCRIPTION
**Pull request by J0**
Relevant Issue: #272

### CODE REVIEW

**Comment by J0** [comment]
Hey @morgo,

Hope that all is well on your end. 

I'm trying to finish the Chinese translation from a while back. Unfortunately, the build seems to fail with the following error after I've edited the file:

image

I've referred to the [relevant section on shortcodes](https://gohugo.io/templates/shortcode-templates/) in the hugo docs and it seems to suggest that the lookup order for shortcodes will be

  1.  `/layouts/shortcodes/<SHORTCODE>.html`
  2.  `/themes/<THEME>/layouts/shortcodes/<SHORTCODE>.html`

The build seems to state that it can't find the shortcode. However, it seems that `/layouts/shortcodes` does contain `pdf.html` just wondering if you have any suggestions as to why hugo can't find the file and how I might fix it.


UPDATE: Apologies, realized that I haven't merged `prod` in for a while. Just need to do that to fix the build.

Thanks!

**Comment by rohit-nayak-ps** [comment]
Hello @J0, I was cleaning up the pending PRs and saw that this has been in Draft for over a year. Just checking if you are still planning further work on this. Were you able to get the build working after the merging with prod? Thanks.

**Comment by J0** [comment]
Hey Rohit,

Hope all is well. Totally forgot about this. Managed to get it built -- thanks for checking :)  I'm not sure how much has changed on the website since the last time I edited the translation. I can get to it and continue translating but it'll take me about 2-3 weeks as I'm still finishing up other projects so if it's urgent and there's someone else who wants to finish it please let them do so.

**Comment by rohit-nayak-ps** [comment]
> on the website since the last time I edited the translation. I can get to it and continue translating but it'll take me about 2-3 weeks as I'm still finishing up other projects so if it's urgent and there's someone else who wants to finish it please let them do so.

Thanks for the quick response. No, there isn't anyone else working on it. Please take your time, I understand that you are busy with your regular work. I will keep this open. Appreciate the effort.

**Comment by J0** [comment]
Okay, thanks for waiting! Just wanted to let you know that I'm coming back to this. Have enlisted two other university schoolmates to help proofread.

File: content/zh/docs/resources/presentations.md [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+
+* [Slack数据库服务概述](https://kccncna19.sched.com/event/UaWu/scaling-resilient-systems-a-journey-into-slacks-database-service-rafael-chacon-guido-iaquinti-slack) - Rafael Chacon & Guido Iaquinti, Slack
+* [如何迁移数据库系统到Vitess](https://kccncna19.sched.com/event/UaiN/how-to-migrate-a-mysql-database-to-vitess-sugu-sougoumarane-morgan-tocker-planetscale) - Sugu Sougoumarane & Morgan Tocker, PlanetScale
+* [在Kubernetes上构建Database as a Service](https://kccncna19.sched.com/event/Uabw/building-a-database-as-a-service-on-kubernetes-abhi-vaidyanatha-lucy-burns-planetscale) - Abhi Vaidyanatha & Lucy Burns, PlanetScale
+* [Vitess: Stateless Storage in the Cloud](https://kccncna19.sched.com/event/UaeD/vitess-stateless-storage-in-the-cloud-sugu-sougoumarane-planetscale) - Sugu Sougoumarane, PlanetScale
+* [Geo-partitioning with Vitess](https://kccncna19.sched.com/event/UagI/geo-partitioning-with-vitess-deepthi-sigireddi-jitendra-vaidya-planetscale) - Deepthi Sigireddi & Jitendra Vaidya, PlanetScale
+* [用Vitess在60分钟内将20TB的字节从AKS迁移到GKE](https://kccncna19.sched.com/event/UaYn/gone-in-60-minutes-migrating-20-tb-from-aks-to-gke-in-an-hour-with-vitess-derek-perkins-nozzle) - Derek Perkins, Nozzle
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: “字节” => “数据”

File: content/zh/docs/resources/presentations.md [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+<iframe src = "/ViewerJS/#../files/2020-lizz-prefosdem.pdf" width='600' height='450' allowfullscreen webkitallowfullscreen></iframe>
+
+## KubeCon圣迭哥2019
+
+KubeCon有关Vitess的演示:
+
+* [Slack数据库服务概述](https://kccncna19.sched.com/event/UaWu/scaling-resilient-systems-a-journey-into-slacks-database-service-rafael-chacon-guido-iaquinti-slack) - Rafael Chacon & Guido Iaquinti, Slack
+* [如何迁移数据库系统到Vitess](https://kccncna19.sched.com/event/UaiN/how-to-migrate-a-mysql-database-to-vitess-sugu-sougoumarane-morgan-tocker-planetscale) - Sugu Sougoumarane & Morgan Tocker, PlanetScale
+* [在Kubernetes上构建Database as a Service](https://kccncna19.sched.com/event/Uabw/building-a-database-as-a-service-on-kubernetes-abhi-vaidyanatha-lucy-burns-planetscale) - Abhi Vaidyanatha & Lucy Burns, PlanetScale
+* [Vitess: Stateless Storage in the Cloud](https://kccncna19.sched.com/event/UaeD/vitess-stateless-storage-in-the-cloud-sugu-sougoumarane-planetscale) - Sugu Sougoumarane, PlanetScale
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Vitess：云端无状态存储

File: content/zh/docs/resources/presentations.md [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+* [Geo-partitioning with Vitess](https://kccncna19.sched.com/event/UagI/geo-partitioning-with-vitess-deepthi-sigireddi-jitendra-vaidya-planetscale) - Deepthi Sigireddi & Jitendra Vaidya, PlanetScale
+* [用Vitess在60分钟内将20TB的字节从AKS迁移到GKE](https://kccncna19.sched.com/event/UaYn/gone-in-60-minutes-migrating-20-tb-from-aks-to-gke-in-an-hour-with-vitess-derek-perkins-nozzle) - Derek Perkins, Nozzle
+
+Vitess也出现在[CNCF](https://www.youtube.com/watch?v=two3TzF9mVY&feature=youtu.be&t=105)的主题演讲中!
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 也曾出现在

File: content/zh/docs/resources/presentations.md [hunk 4]
```diff
@@ -0,0 +1,216 @@
...
+
+## Highload 2019
+
+Sugu Sougoumarane 在莫斯科的Highload会议介绍Vitess。
+
+{{< pdf src="/ViewerJS/#../files/2019-sugu-highload.pdf" >}}
+
+## Utah Kubernetes 2019 聚会
+
+Jiten Vaidya展示了如何扩展Vitess来创建 jurisdiction-aware数据库集群。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Jiten Vaidya展示了如何通过扩展Vitess来创建 jurisdiction-aware 的数据库集群。

File: content/zh/docs/resources/presentations.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+
+{{< pdf src="/ViewerJS/#../files/2019-jiten-utah.pdf" >}}
+
+## CNCF 2019 巴黎聚会
+
+
+
+Sugu Sougoumarane和Morgan Tocker主持了一个关于如何在Kubernetes上使用Vitess的三个小时的演讲。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Sugu Sougoumarane和Morgan Tocker针对如何在Kubernetes上使用Vitess进行了时长三个小时的演讲。

File: content/zh/docs/resources/presentations.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+**使用Vitess的前90天**
+
+Morgan Tocker 讲述了他与Vitess的经历。
+
+{{< pdf src="/ViewerJS/#../files/2019-morgan-percona-eu.pdf" >}}
+
+
+**在Kubernetes上运行分片MySQL**
+
+Sugu Sougoumarane介绍了如何在Kubernetes上运行切分MySQL。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 分片MySQL

File: content/zh/docs/resources/presentations.md [hunk 7]
```diff
@@ -0,0 +1,216 @@
...
+
+{{< pdf src="/ViewerJS/#../files/2019-sugu-percona-eu.pdf" >}}
+
+## Vitess Meetup 2019 @ Slack HQ
+
+**Vitess: 新和即将到来的功能**
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Vitess：全新功能及未来展望

File: content/zh/docs/resources/presentations.md [hunk 8]
```diff
@@ -0,0 +1,216 @@
...
+
+Deepthi Sigireddi分享了Vitess的新特性，以及未来的路线图。
+
+{{< pdf src="/ViewerJS/#../files/2019-deepthi-vitess-meetup.pdf" >}}
+
+
+**Deploying multi-cell Vitess**
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 部署multi-cell Vitess

File: content/zh/docs/resources/presentations.md [hunk 9]
```diff
@@ -0,0 +1,216 @@
...
+
+Rafael Chacon Vivas 介绍Slack如何使用Vitess.
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Rafael Chacon Vivas 介绍了Vitess是如何在Slack中被运用的。

File: content/zh/docs/resources/presentations.md [hunk 10]
```diff
@@ -0,0 +1,216 @@
...
+
+{{< pdf src="/ViewerJS/#../files/2019-rafael-vitess-meetup.pdf" >}}
+
+
+**Pinterest如何使用vitess**
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Pinterest中的vitess

File: content/zh/docs/resources/presentations.md [hunk 11]
```diff
@@ -0,0 +1,216 @@
...
+
+David Weitzman 介绍Pinterest如何使用Vitess.
+
+{{< youtube id="1cWWlaqlia8" autoplay="false" >}}
+
+
+**后悔也来不及**
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 不必再后悔

File: content/zh/docs/resources/presentations.md [hunk 12]
```diff
@@ -0,0 +1,216 @@
...
+
+Sugu Sougoumarane 演示VReplication的最新功能.
+
+{{< youtube id="B1Nrtptjtcs" autoplay="false" >}}
+
+## Cloud Native Show 2019
+
+**Vitess at scale - how Nozzle.io runs MySQL on Kubernetes**
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 大规模使用Vitess - Nozzle.io如何在Kubernetes上运行MySQL

File: content/zh/docs/resources/presentations.md [hunk 13]
```diff
@@ -0,0 +1,216 @@
...
+
+Derek Perkins 介绍Nozzle如何使用Vitess。
+
+[Listen to Podcast](https://anchor.fm/cloud-native-show/episodes/Vitess-at-scale---how-Nozzle-io-runs-MySQL-on-Kubernetes-e4m5lo)
+
+## CNCF Webinar 2019
+
+**Vitess: Sharded MySQL on Kubernetes**
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Vitess: Kubernetes 上的分片MySQL

File: content/zh/docs/resources/presentations.md [hunk 14]
```diff
@@ -0,0 +1,216 @@
...
+
+Sugu Sougoumarane为Kubernetes使用者介绍Vitess。
+
+{{< youtube id="E6H4bgJ3Z6c" autoplay="false" >}}
+
+## Kubecon 2019中国
+
+**京东如何运营全球最大的Vitess集群**
+
+Xuhaihua 和 Jin Ke Xie使用世界上最大的Vitess数据库群集。他们呈现两年内学到的知识。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Xuhaihua 和 Jin Ke Xie 分享了他们两年来在运行世界上最大的Vitess数据库集群中学到的经验。

File: content/zh/docs/resources/presentations.md [hunk 15]
```diff
@@ -0,0 +1,216 @@
...
+
+{{< youtube id="qww4UVNG3Io" autoplay="false" >}}
+
+## RootConf 2019
+
+**OLTP or OLAP: why not both?**
+
+PlanetScale的Jiten Vaidya解释如何在Vitess同时使用OLTP和OLAP。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 如何在Vitess上同时使用

File: content/zh/docs/resources/presentations.md [hunk 16]
```diff
@@ -0,0 +1,216 @@
...
+
+{{< youtube id="bhzJJF82mFc" autoplay="false" >}}
+
+## Kubecon 19巴塞罗那
+
+**Vitess Deep Dive**
+
+PlanetScale的Jiten Vaidya和Dan Kozlowski详细的介绍Vitess.
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 详细地介绍了

File: content/zh/docs/resources/presentations.md [hunk 17]
```diff
@@ -0,0 +1,216 @@
...
+**Vitess: 在Kubernetes上运行分片MySQL**
+
+Sugu Sougoumarane 介绍如何在k8s上跑分片MySQL。
+
+{{< youtube id="v7oxiVmGXp4" autoplay="false" >}}
+
+
+**MySQL, Kubernetes, 商业和企业**
+
+David Cohen (Intel), Steve Shaw (Intel) 和 Jiten Vaidya (PlanetScale) 讨论开源原生云的数据库
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 开源云原生的数据库

File: content/zh/docs/resources/presentations.md [hunk 18]
```diff
@@ -0,0 +1,216 @@
...
+
+[View Talk Abstract and Slides](https://www.percona.com/live/19/sessions/an-open-source-cloud-native-database-cndb)
+
+## Velocity 2018 纽约
+
+**Smooth scaling: Slack’s journey toward a new database**
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 平稳的规模扩展：Slack追寻全新数据库的道路

File: content/zh/docs/resources/presentations.md [hunk 19]
```diff
@@ -0,0 +1,216 @@
...
+
+
+作为一个年轻的公司，Slack经历了巨大的增长，每周服务超过900万活跃客户。但随着经济的增长，随之而来的是更大的增长痛苦。Slack在过去几年的快速增长超过了其原始的sharded MySQL数据库的扩展能力，这对公司的客户和工程师造成了负面影响。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 但随着经济的增长，随之而来的是更大的增长痛苦。
但规模的扩展也带来了更多其他方面的烦恼。

File: content/zh/docs/resources/presentations.md [hunk 20]
```diff
@@ -0,0 +1,216 @@
...
+
+
+Ameet Kotian解释了一小群工程师是如何开始寻找正确的数据库解决方案的，最终他们找到了Vitess，一个强大的MySQL开源数据库集群解决方案。Vitess结合了MySQL的特性和NoSQL数据库的可伸缩性。它多年来一直为Youtube提供流量服务，并拥有强大的社区。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Ameet Kotian解释了一小群工程师是如何开始寻找正确的数据库解决方案的
Ameet Kotian讲述了几名工程师如何开始寻找正确的数据库解决方案的故事。最终...

File: content/zh/docs/resources/presentations.md [hunk 21]
```diff
@@ -0,0 +1,216 @@
...
+
+
+尽管Vitess满足了Slack的很多需求，但它并不是一个开箱即用的解决方案。Ameet分享了在面对堆叠操作挑战(如AWS问题、MySQL复制、自动故障转移、部署策略等)时，如何在几乎不影响客户的情况下计划和执行Vitess的旅程。Ameet还涵盖了Vitess的架构、权衡，以及在Slack下Vitess的未来会是什么样子。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Ameet分享了在面对堆叠操作挑战
Ameet分享了在面对成堆的运营挑战

File: content/zh/docs/resources/presentations.md [hunk 22]
```diff
@@ -0,0 +1,216 @@
...
+
+
+Slack的高级存储操作工程师Ameet Kotkian向我们展示了Slack是如何使用Vitess的。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 高级存储运营工程师

File: content/zh/docs/resources/presentations.md [hunk 23]
```diff
@@ -0,0 +1,216 @@
...
+
+<iframe src = "/ViewerJS/#../files/20181002-ameet-velocity-slides.pdf" width='600' height='450' allowfullscreen webkitallowfullscreen></iframe>
+
+## Percona Live 2017 欧洲
+
+**Slack迁移到vitess的过程**
+
+Slack正在对MySQL基础设施进行重大迁移，以使用Vitess灵活的分片和管理，而不是我们简单的基于应用程序的分片路由和手动管理。这种努力是由架构的需求驱动的，该架构可扩展以满足我们最大客户不断增长的需求，并在压力下保持稳定和高性能的服务，每小时执行数十亿的MySQL事务。这次演讲将展示变革背后的动机，为什么Vitess胜出成为最佳选择，以及我们如何为变革奠定基础。最后，我们将讨论在我们最初的迁移工作中发现的一些挑战和意外(好的和坏的)，并提出一些改进Vitess生态系统的方法，以帮助未来的迁移工作。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 以使用Vitess灵活的分片和管理，而不是我们简单的基于应用程序的分片路由和手动管理
以使用Vitess灵活的分片和管理来取代简单的基于应用程序的分片路由和手动管理

该架构可扩展以满足我们最大客户不断增长的需求
该架构可扩展以满足我们巨大用户量和功能上日益增常的需求

File: content/zh/docs/resources/presentations.md [hunk 24]
```diff
@@ -0,0 +1,216 @@
...
+
+Michael Demmer在[Percona Live Europe 2017](https://www.percona.com/live/e17/sessions/migrating-to-vitess-at-slack-scale)告诉我们他们是如何迁移到Vitess。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 如何迁移到Vitess的

File: content/zh/docs/resources/presentations.md [hunk 25]
```diff
@@ -0,0 +1,216 @@
...
+## Percona Live 2016
+
+[Sugu](https://github.com/sougou)和[Anthony](https://github.com/enisoc) 展示 如何使用新版本的 Vitess。在新版本中，keyspace id对应用程序是隐藏的。 他们现场演示了如何重新分片一个留声簿应用程序，并解释了VTGate的新功能是如何实现这一点的。
+
+{{< pdf src="/ViewerJS/#../files/percona-2016.pdf" >}}
+
+## CoreOS January 2016聚会
+
+Vitess队员[Anthony Yeh](https://github.com/enisoc)'s talk at
+the [January 2016 CoreOS Meetup](http://www.meetup.com/coreos/events/228233948/)
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Vitess成员[Anthony Yeh](https://github.com/enisoc) 在
[January 2016 CoreOS Meetup](http://www.meetup.com/coreos/events/228233948/) 中

File: content/zh/docs/resources/presentations.md [hunk 26]
```diff
@@ -0,0 +1,216 @@
...
+讨论了在Kubernetes中运行分布式数据库的挑战和技术，然后深入探讨了[Vitess在Kubernetes](https://github.com/vitessio/vitess/tree/master/examples/kubernetes)部署模板上的设计权衡。
+
+{{< pdf src="/ViewerJS/#../files/coreos-meetup-2016-01-27.pdf" >}}
+
+## Oracle OpenWorld 2015
+
+Vitess队员[Anthony Yeh](https://github.com/enisoc)'s 在Oracle OpenWorld 2015讨论了[云本地](http://cncf.io) 范式如何影响MySQL在Cloud的使用.这次演讲还包括了对[transparent, live resharding](../../sharding)的深入探讨，这是关键之一
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Vitess成员[Anthony Yeh](https://github.com/enisoc) 在Oracle OpenWorld 2015中讨论了...

File: content/zh/docs/resources/presentations.md [hunk 27]
```diff
@@ -0,0 +1,216 @@
...
+Vitess的特性使其能够很好地适应原生云的环境.
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: 探讨，这是让Vitess能够良好适应云原生环境的关键功能之一。

File: content/zh/docs/resources/presentations.md [hunk 28]
```diff
@@ -0,0 +1,216 @@
...
+
+{{< pdf src="/ViewerJS/#../files/openworld-2015-vitess.pdf" >}}
+
+## Percona Live 2015
+
+Vitess队员[Anthony Yeh](https://github.com/enisoc)'s 2015年Percona Live上的演讲提供了Vitess的概述 2015年Percona Live提供了Vitess的概述，并解释了Vitess是如何与Kubernetes和Docker一起生活在一个集装箱化的世界里。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Vitess成员[Anthony Yeh](https://github.com/enisoc)在2015年Percona Live上的演讲提供了针对Vitess的概述，并解释了Vitess是如何通过Kubernetes和Docker实现容器化的。

File: content/zh/docs/resources/presentations.md [hunk 29]
```diff
@@ -0,0 +1,216 @@
...
+
+{{< pdf src="/ViewerJS/#../files/percona-2015-vitess-and-kubernetes.pdf" >}}
+
+## Google I/O 2014 - Scaling with Go: YouTube's Vitess
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Google I/O 2014 - 用Go语言实现扩展 - YouTube中的Vitess

Scaling with Go: YouTube's Vitess

File: content/zh/docs/resources/presentations.md [hunk 30]
```diff
@@ -0,0 +1,216 @@
...
+
+Vitess队员Sugu Sougoumarane](https://github.com/sougou) 谈论Vitess如何解决YouTube的可伸缩性问题，以及如何使用Go实现更好的可伸缩性的技巧。<br><br>
```
Comment (yuntongzhang) [inline content/zh/docs/resources/presentations.md]: Vitess成员[Sugu Sougoumarane](https://github.com/sougou) 谈论Vitess如何解决YouTube的可扩展性问题，以及在使用Go语言实现可扩展性时的一些技巧。

File: content/zh/docs/resources/roadmap.md [hunk 31]
```diff
@@ -0,0 +1,32 @@
+---
+title: Vitess 路线图
+description: Upcoming features planned for development
+weight: 2
+---
+
+Vitess以开源方式发布。Vitess是由贡献者开发。许多贡献者用Vitess在生产环境中。他们在Vitess上增添功能来解决自己的问题。 因次我们不能保证以下功能将按固定顺序实现。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/roadmap.md]: 许多贡献者用Vitess在生产环境中。他们在Vitess上增添功能来解决自己的问题。
许多贡献者在生产环境中使用Vitess，并且在Vitess上增添功能来解决自己的问题。

File: content/zh/docs/resources/roadmap.md [hunk 32]
```diff
@@ -0,0 +1,32 @@
...
+
+{{< info >}}
+如果您对路线图有特点的问题， 我们建议您在 [Slack channel](https://vitess.slack.com)发布您的问题。您可点击在右上角的图标加入Slack。 我们的Slack群非常活跃，群员非常热亲。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/roadmap.md]: 如果您对路线图有特点的问题， 
如果您对路线图有特定的问题，

群员非常热亲。
群员非常热情。

File: content/zh/docs/resources/roadmap.md [hunk 33]
```diff
@@ -0,0 +1,32 @@
...
+- 采用一个[一致的发布周期](https://github.com/vitessio/enhancements/blob/master/veps/vep-1.md)
+- 提高易用性
+
+## 中期
+
+- VReplication
+  - 支持模式改变
+  - 回填查找索引
+  - 支持数据迁移
+- 拓扑服务: 减少拓扑服务的依赖项。i.e. 即使拓扑服务中断几个小时，我们还得正常使用Vitess。 拓扑服务  只适用于 Passive Discovery.
```
Comment (yuntongzhang) [inline content/zh/docs/resources/roadmap.md]: 减少拓扑服务的依赖项
减少对拓扑服务的依赖

即使拓扑服务中断几个小时，我们还得正常使用Vitess
即使拓扑服务中断几个小时，Vitess仍需要正常运行

拓扑服务  只适用于 Passive Discovery.
拓扑服务只应该在Passive Discovery中被使用。

File: content/zh/docs/resources/roadmap.md [hunk 34]
```diff
@@ -0,0 +1,32 @@
...
+- 支持PostgreSQL： Vitess 应该支持 PostgreSQL来存储数据以及在VTGate用Postgres Protocol。
```
Comment (yuntongzhang) [inline content/zh/docs/resources/roadmap.md]: Vitess 应该支持 PostgreSQL来存储数据以及在VTGate用Postgres Protocol。
Vitess 应支持通过PostgreSQL来存储数据，并且让PostgreSQL能够通过VTGate的协议实现沟通。

**Comment by J0** [comment]
Thanks Yuntong! <3

**Comment by J0** [comment]
Hey @rohit-nayak-ps,
 
Hope you're having a good weekend. With the help of @yuntongzhang I think we have reached a state where this PR is ready for review. I will ping the owner of the issue so that he can review but just thought I'd let you know as well.

Thanks for your patience!

File: content/zh/docs/resources/presentations.md [hunk 35]
```diff
@@ -0,0 +1,204 @@
...
+
+## Velocity 2018 纽约
+
+**平稳的规模扩展：Slack 追寻全新数据库的道路**
+
+作为一个年轻的公司，Slack 经历了巨大的增长，每周服务超过 900 万活跃客户。但规模的扩展也带来了更多其他方面的烦恼。Slack 在过去几年的快速增长超过了其原始的 sharded MySQL 数据库的扩展能力，这对公司的客户和工程师造成了负面影响。
+
+Ameet Kotian 讲述了几名工程师如何开始寻找正确的数据库解决方案的故事。最终他们找到了 Vitess，一个强大的 MySQL 开源数据库集群解决方案。Vitess 结合了 MySQL 的特性和 NoSQL 数据库的可伸缩性。它多年来一直为 Youtube 提供流量服务，并拥有强大的社区。
+
+尽管 Vitess 满足了 Slack 的很多需求，但它并不是一个开箱即用的解决方案。Ameet 分享了在面对成堆的运营挑战(如 AWS 问题、MySQL 复制、自动故障转移、部署策略等)时，如何在几乎不影响客户的情况下计划和执行 Vitess 的旅程。Ameet 还涵盖了 Vitess 的架构、权衡，以及在 Slack 下 Vitess 的未来会是什么样子。
```
Comment (inolddays) [inline content/zh/docs/resources/presentations.md]: 如何在尽量不影响客户业务的情况下，如何测试和落地vitess之旅。Ameet还谈到了vitess的整体架构，设计上的取舍和权衡，以及vitess在Slack的特殊业务场景下，未来会成长成的样子。

File: content/zh/docs/resources/presentations.md [hunk 36]
```diff
@@ -0,0 +1,204 @@
...
+
+Sugu Sougoumarane 为 Kubernetes 使用者介绍 Vitess。
+
+{{< youtube id="E6H4bgJ3Z6c" autoplay="false" >}}
+
+## Kubecon 2019 中国
+
+**京东如何运营全球最大的 Vitess 集群**
+
+Xuhaihua 和 Jin Ke Xie 分享了他们两年来在运行世界上最大的 Vitess 数据库集群中学到的经验。
```
Comment (inolddays) [inline content/zh/docs/resources/presentations.md]: 许海华和谢金科分享了两年多来，京东运行世界上最大规模Vitess 数据库集群的心得和体会。

File: content/zh/docs/resources/roadmap.md [hunk 37]
```diff
@@ -0,0 +1,32 @@
...
+- 提高易用性
+
+## 中期
+
+- VReplication
+  - 支持模式改变
+  - 回填查找索引
+  - 支持数据迁移
+- 拓扑服务:减少对拓扑服务的依赖。i.e. 即使拓扑服务中断几个小时，Vitess 仍需要正常运行。 拓扑服务只应该在 Passive Discovery 中被使用。
+- Vitess 应支持通过 PostgreSQL 来存储数据，并且让 PostgreSQL 能够通过 VTGate 的协议实现沟通。
```
Comment (inolddays) [inline content/zh/docs/resources/roadmap.md]: - 拓扑服务:减少对拓扑服务的依赖。例如 :即使拓扑服务中断几个小时，Vitess 需要正常运行。 拓扑服务只用于被动发现。
- Vitess 支持 后端使用PostgreSQL 存储数据， vtgate 支持 PG 协议。

File: content/zh/docs/resources/roadmap.md [hunk 38]
```diff
@@ -0,0 +1,32 @@
...
+Vitess 以开源方式发布。Vitess 是由贡献者开发。许多贡献者在生产环境中使用 Vitess，并且在 Vitess 上增添功能来解决自己的问题。 因次我们不能保证以下功能将按固定顺序实现。
+
+{{< info >}}
+如果您对路线图有特定的问题， 我们建议您在 [Slack channel](https://vitess.slack.com)发布您的问题。您可点击在右上角的图标加入 Slack。 我们的 Slack 群非常活跃，群员非常热情。
+{{< /info >}}
+
+## 短期
+
+- VReplication
+  - 改进重新分片的工作流程。提高它的灵活性，速度和可靠性。
```
Comment (inolddays) [inline content/zh/docs/resources/roadmap.md]: - 改进拆分的工作流。提高工作流的速度、灵活性和可靠性。

File: content/zh/docs/resources/presentations.md [hunk 39]
```diff
@@ -0,0 +1,204 @@
...
+
+{{< pdf src="/ViewerJS/#../files/2019-rafael-vitess-meetup.pdf" >}}
+
+**Pinterest 中的 Vitess**
+
+David Weitzman 介绍 Pinterest 如何使用 Vitess.
+
+{{< youtube id="1cWWlaqlia8" autoplay="false" >}}
+
+**不必再后悔**
```
Comment (inolddays) [inline content/zh/docs/resources/presentations.md]: 可能翻译成“不再留下遗憾”会更合适一些？

**Comment by inolddays** [comment]
LGTM

**Comment by inolddays** [comment]
@J0   @rohit-nayak-ps   seems all are ok now ,  i am not sure how could i merge this ? or @rohit-nayak-ps  can help to merge it to ch branch ?

**Comment by J0** [comment]
@Johnny-Three Thanks for the review! I'm not too sure how to get this merged either -- will have to check with @rohit-nayak-ps . I think @askdba might be managing the site now. Not too sure.

**Review by askdba — APPROVED** [review]
LGTM

**Pull request merged**